docs/configuring-output-after.asciidoc

```diff
@@ -3,7 +3,9 @@
 [float]
 === Sourcemaps
 
-Sourcemaps can be uploaded through various outputs but can currently only be applied when Elasticsearch is the output.
+Sourcemaps can be uploaded through all outputs but must eventually be stored in Elasticsearch.
+When using outputs other than Elasticsearch, `source_mapping.elasticsearch` must be set for source maps to be applied.
+See <<config-sourcemapping-elasticsearch>> for more details.
```

Comment:
can you add a note that then you also need to change source_mapping.index_pattern to whatever matches the used output? (also in the yml file)

Reply:
good call, done in the docs.  The config file has good language around this.